js/main.js

```diff
@@ -0,0 +1,32 @@
+var $saveButton = document.querySelector('.save-button');
+var $title = document.querySelector('#title');
+var $photo = document.querySelector('#photo');
+var $note = document.querySelector('#note-input');
+var $img = document.querySelector('img');
+
+function saveAction(event) {
+  event.preventDefault();
+  data.entries.push({});
+  data.entries[data.nextEntryId - 1].imgSrc = $photo.value;
+  data.entries[data.nextEntryId - 1].title = $title.value;
+  data.entries[data.nextEntryId - 1].notes = $note.value;
```

Comment:
I would just make the object first then push it so that this syntax can be simplified.